ns1/rest/zones.py

```diff
@@ -110,6 +110,71 @@ def search(
             errback=errback,
         )
 
+    def list_versions(
+            self,
+            zone,
+            callback=None,
+            errback=None
+    ):
+        request = "{}/{}/versions".format(self.ROOT, zone)
+        return self._make_request(
+            "GET",
+            request,
+            params={},
+            callback=callback,
+            errback=errback,
+        )
+
+    def create_version(
+            self,
+            zone,
+            force=False,
+            callback=None,
+            errback=None):
+        request = "{}/{}/versions?force={}".format(
+            self.ROOT, zone, str.lower(str(force))
+        )
```

Comment:
Similar to above, we _could_ use:

```suggestion
        request = f"{self.ROOT}/{zone}/versions?force={str(force).lower()}"
```